scripts/joinmarket-qt.py

```diff
@@ -2005,13 +2006,19 @@ def loadWalletFromBlockchain(self, firstarg=None, pwd=None):
         self.wallet_service.autofreeze_warning_cb = self.autofreeze_warning_cb
         self.wallet_service.startService()
         self.syncmsg = ""
-        self.walletRefresh = task.LoopingCall(self.updateWalletInfo)
+        self.walletRefresh = task.LoopingCall(self.updateWalletInfo, None, None)
         self.walletRefresh.start(5.0)
```

Comment:
Do we want to consider starting `walletRefresh` immediately instead of waiting for 5 seconds?

Reply:
I have a vague memory that something weird happened if it fires immediately, but I could be wrong. But in any case, there's no chance that the wallet sync has finished at 0.0 seconds here, so it's probably academic.
You could certainly argue 2.0 seconds makes more sense, but shrug, not  too bothered about that.